core/settings/base.py

```diff
@@ -33,6 +33,8 @@
     "drf_yasg",
     # app imports
     "users",
+    "articles",
+    "taggit",
```

Comment:
third party imports should be  under drf_yasg, app imports are for apps created

Reply:
moved